src/test/resources/org/broadinstitute/hellbender/tools/haplotypecaller/expected.testForceCallingNotProducingNoCalls.gatk4.vcf

```diff
@@ -47,6 +47,7 @@
 20	10003832	.	G	A	2720.06	.	AC=2;AF=1.00;AN=2;DP=67;ExcessHet=0.0000;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=56.82;QD=29.03;SOR=1.473	GT:AD:DP:GQ:PL	1/1:0,66:66:99:2734,199,0
 20	10004094	.	A	C,T	1840.06	.	AC=2,0;AF=1.00,0.00;AN=2;DP=55;ExcessHet=0.0000;FS=0.000;MLEAC=2,0;MLEAF=1.00,0.00;MQ=52.62;QD=34.72;SOR=1.524	GT:AD:DP:GQ:PL	1/1:0,53,0:53:99:1854,157,0,1899,164,2091
 20	10004147	.	A	G	1852.06	.	AC=2;AF=1.00;AN=2;DP=54;ExcessHet=0.0000;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=55.19;QD=30.67;SOR=1.136	GT:AD:DP:GQ:PL	1/1:0,52:52:99:1866,156,0
+20	10004193	.	G	T	0	LowQual	AC=1;AF=0.500;AN=2;BaseQRankSum=-0.392;DP=54;ExcessHet=0.0000;FS=2.369;MLEAC=1;MLEAF=0.500;MQ=59.14;MQRankSum=3.385;QD=0.00;ReadPosRankSum=1.724;SOR=0.941	GT:AD:DP:GQ:PL	0/1:23,28:51:99:945,0,776
```

Comment:
do you have a reason for this one? This seems to be more dangerous

Reply:
Just a difference in a marginal haplotype down the list being kept or not.  With a QUAL of 0 it's not going to affect the calls.